CMakeRC.cmake

```diff
@@ -585,7 +585,7 @@ function(cmrc_add_resources name)
         endif()
         get_filename_component(dirpath "${ARG_PREFIX}${relpath}" DIRECTORY)
         _cmrc_register_dirs("${name}" "${dirpath}")
-        get_filename_component(abs_out "${libdir}/intermediate/${relpath}.cpp" ABSOLUTE)
+        get_filename_component(abs_out "${libdir}/intermediate/${ARG_PREFIX}${relpath}.cpp" ABSOLUTE)
```

Comment:
`ARG_PREFIX` is garantied to end with `/` from the if 3 lines above: https://github.com/vector-of-bool/cmrc/blob/a64bea50c05594c8e7cf1f08e441bb9507742e2e/CMakeRC.cmake#L583-L585

Further, `ARG_PREFIX` may is empty if no prefix is set, so an explicit extra `/` would lead to a duplicate `/` in that case. The same pattern is already used in lines 586 and 610, so should be consistent with that.

Reply:
Good catch.